group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilderTest.java

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
Should we extend the tests in this class to cover the new method/logic in the builder?